lib/beaker/tasks/quick_start.rb

```diff
@@ -1,49 +1,85 @@
 require 'beaker-hostgenerator'
 
+VAGRANT  = BeakerHostGenerator::CLI.new(['ubuntu1404-64default.mdcal', '--hypervisor=vagrant',
+                                         '--global-config={box_url=https://vagrantcloud.com/puppetlabs/boxes/ubuntu-14.04-64-nocm,box=puppetlabs/ubuntu-14.04-64-nocm'])
+VMPOOLER = BeakerHostGenerator::CLI.new(['redhat7-64default.mdcal-redhat7-64af'])
+
 namespace :beaker_quickstart do
 
-  desc 'Generate Default Beaker Host Config File'
-  task :gen_hosts do
-    cli = BeakerHostGenerator::CLI.new(['redhat7-64default.mdcal-redhat7-64af'])
+  desc 'Generate Default Beaker Host Config File, valid options are: vmpooler or vagrant.'
+  task :gen_hosts, [:provisioner] do |t, args|
+    hosts_file = "acceptance/config/default__#{args[:provisioner]}_hosts.yaml"
+    if args[:provisioner] == 'vagrant'
+      cli = VAGRANT
+    elsif args[:provisioner] == 'vmpooler'
+      cli = VMPOOLER
+    else
+      puts "No provisioner provided, defaulting to vmpooler."
+      cli = VMPOOLER
```

Comment:
i think we might want to move to defaulting to what users outside of puppet need and use; especially in the docs/guides.  those of us inside puppet have plenty of resources to learn how to use vmpooler with beaker.  

thoughts?